src/routes/sessions/[uuid]/.Requester/.Requester.svelte

```diff
@@ -3,46 +3,45 @@
   import Notification from '$components/Notification/Notification.svelte';
   import { NOTIFICATION_TYPES, NOTIFICATION_VALUES } from '$components/Notification/enums';
   import { DEVICE_CALL_QUEUE_SETUP_GUIDE } from '$lib/constants';
-
+  import customerSupport from '$lib/static/img/customer-support.svg';
   export let demo: JSON;
   export let embeddable: boolean;
 
   const { isSIP, isSDK, isIC, uuid } = demo;
 
-  let sipQueues = [{ title: 'Looking for Assistance?', extensionNumber: 1111 }];
+  let sipQueues = [{ title: 'Looking for Assistance?', extensionNumber: 1111, sipImage: customerSupport }];
+  let extensionNumber = 1111;
 
   if (demo.isSIP) {
     sipQueues = [
-      { title: demo.sipTitle || 'Looking for Assistance?', extensionNumber: demo.extensionNumber },
-      demo.sipTitle1 && { title: demo.sipTitle1, extensionNumber: demo.extensionNumber1 },
-      demo.sipTitle2 && { title: demo.sipTitle2, extensionNumber: demo.extensionNumber2 },
-      demo.sipTitle3 && { title: demo.sipTitle3, extensionNumber: demo.extensionNumber3 }
+      {
+        title: demo.sipTitle1 || 'Looking for Assistance?',
```

Comment:
Here and at other places, for checking null/missing value, we can use the nullish coalescing (` ??` ) operator.

Reply:
`??` does not work here but I moved the logic to the server-side.